src/blockchain/Blockchain.ts

```diff
@@ -112,7 +105,7 @@ export default class Blockchain implements IBlockchainApi {
             log.warn(`Extrinsic error ocurred: ${extrinsicError}`)
             reject(extrinsicError)
           }
-          if (status.type === 'Finalised') {
+          if (status.type === 'Finalized') {
```

Comment:
Suggestion for a follow-up ticket: use consts or Polkadot methods for the statuses over freetext strings